site/source/docs/building_from_source/building_fastcomp_manually_from_source.rst

```diff
@@ -64,7 +64,7 @@ To build the Fastcomp code from source:
 
   ::
 
-    cmake .. -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
+    cmake ../src -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
```

Comment:
this change is unneeded I think - when not using the emsdk, there is no `src/` dir.